react-site/src/Navbar/Navbar.jsx

```diff
@@ -1,19 +0,0 @@
-import * as React from 'react';
```

Comment:
Why do we not have a navbar anymore?

Reply:
I think we could have the navbar component without the routes implemented. Right now the navigation is a little awkward, because if the user clicks anywhere near the top of the screen they get taken back to the homepage.

Reply:
happy to reimplement this, and if the clicking is an issue i can change the styling as well! also feel free to push commits to the branch to change the navbar (it's a new component now in `components/`)